### PR TITLE
psuedo-init: Do not execute with set -x.

### PR DESCRIPTION
**Pull request by smoser**
set -x just makes clogs good logs (what we strive for) with
obnoxious noise.

Also, move a debug statement a bit earlier (before mount of /tmp)
which could fail.

### CODE REVIEW

File: scripts/psuedo-init [hunk 1]
```diff
@@ -149,7 +149,7 @@ wait_for_ready() {
 main() {
     local short_opts="h:v"
     local long_opts="help,network:,verbose"
-    local getopt_out="" args="$*"
+    local getopt_out="" args="$*" argn="$#"
```
Comment (raharper) [inline scripts/psuedo-init:152]: This seems unrelated unless you also wanted to augment the debug print before the mount?

Comment (smoser) [reply scripts/psuedo-init:152]: yeah. i had meant to also log argn. fixing.

Comment (smoser) [reply scripts/psuedo-init:152]: added argn to the debug.

**Review by raharper — APPROVED** [review]
